### ISSUE TITLE
collision bug

### ISSUE THREAD

**Issue by obiima** [iwanders/MFRC630#1]
I found a collision bug.

```
      // set it to insert zeros in case of a collision, (1<<7)
      // We want to shift the bits with RxAlign
      uint8_t rxalign = known_bits % 8;
      MFRC630_PRINTF("Setting rx align to: %hhd\n", rxalign);
      mfrc630_write_reg(MFRC630_REG_RXBITCTRL, (1<<7) | (rxalign<<4));
```

should be

```
      // set it to insert zeros in case of a collision, (0<<7)
      // We want to shift the bits with RxAlign
      uint8_t rxalign = known_bits % 8;
      MFRC630_PRINTF("Setting rx align to: %hhd\n", rxalign);
      mfrc630_write_reg(MFRC630_REG_RXBITCTRL, (0<<7) | (rxalign<<4));
```

**Comment by iwanders** [comment]
Hi Obiima, thanks for reporting this. It is very likely you are correct on this matter, since we do want to set the bits after the collision to zeros.

I don't have access to hardware at the moment to test this, I presume you have tested this thoroughly and can confirm that this increases the reliability of the collision handling?

Would you like to create a pull request for your fix or do you want me to incorporate your improvement?

**Comment by iwanders** [comment]
Decided to go forward with the fix, I've incorporated your bug fix in #2. I'm grateful you took the time to look into this bug and proposed a fix.
